aiohttp/http_parser.py

```diff
@@ -766,6 +771,9 @@ def __init__(
         self._chunk_tail = b""
         self._auto_decompress = auto_decompress
         self._lax = lax
+        self._headers_parser = headers_parser
+        # HeadersParser expects status/request line first, so skips the first line.
+        self._trailer_lines: list[bytes] = [b""]
```

Comment:
This feels rather awkward. Maybe we need to rework HeadersParser slightly? Maybe we just pass `lines[1:]` in the request/response parsers and stop HeadersParser skipping the first line?